.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        # Run tests across all platforms, to ensure developers
+        # can use any environment to run basic tests.
+        platform: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
I'm not sure I understand why we need to run these tests on **pull requests** on macOS and Windows. This isn't really for local tests and their environments. Is apiserver likely do be deployed on macOS and Windows?

Reply:
I'll make it only run for ubuntu-latest for now.